core/src/main/scala/water/api/RDDs/RDDsHandler.scala

```diff
@@ -75,7 +75,7 @@ class RDDsHandler(val sc: SparkContext, val h2oContext: H2OContext) extends Hand
       throw new H2ONotFoundArgumentException(s"RDD with ID '${s.rdd_id}' does not exist, can not proceed with the transformation!")
     }
     // TODO(vlad): take care of the cases when the data are missing
-    val rdd = sc.getPersistentRDDs.get(s.rdd_id).get
+    val rdd = sc.getPersistentRDDs(s.rdd_id)
     val h2oFrame = if(s.h2oframe_id == null) convertToH2OFrame(rdd, None) else convertToH2OFrame(rdd,Some(s.h2oframe_id.toLowerCase))
```

Comment:
this can _probably_ be rewrtitten as
val h2oFrame = convertToH2OFrame(rdd, Option(s.h2oframe_id) map (_.toLowerCase))